dvc/repo/init.py

```diff
@@ -108,8 +110,10 @@ def init(root_dir=os.curdir, no_scm=False, force=False, subdir=False):
 
     if scm.ignore_file:
         scm.add([os.path.join(dvc_dir, scm.ignore_file)])
-        logger.info("\nYou can now commit the changes to git.\n")
+        if not quiet:
+            logger.info("\nYou can now commit the changes to git.\n")
 
-    _welcome_message()
+    if not quiet:
+        _welcome_message()
```

Comment:
How about we move it to `dvc/command/init`? Seems like these messages don't belong to API, likely an old leftover.